src/viur/core/skeleton.py

```diff
@@ -1132,8 +1173,11 @@ def txnUpdate(write_skel):
             db_obj = skel.preProcessSerializedData(db_obj)
 
             # Allow the custom DB Adapter to apply last minute changes to the object
-            if skel.customDatabaseAdapter:
-                db_obj = skel.customDatabaseAdapter.preprocessEntry(db_obj, skel, change_list, is_add)
+            for adapter in utils.ensure_iterable(skel.customDatabaseAdapter):
```

Comment:
Wouldn't it be more efficient to create / normalize an iterable once in the meta class or similar instead of doing this every time at runtime when saving?!

Reply:
@sveneberth I've achieved your suggestion in commit 0071940f7fe0435c3df2091994f99e946805cc48